app/src/main/java/com/android/voyageur/ui/trip/Tabs.kt

```diff
@@ -28,13 +29,19 @@ fun TopTabs(tripsViewModel: TripsViewModel, navigationActions: NavigationActions
   // Remember the currently selected tab index
   var selectedTabIndex by remember { mutableIntStateOf(0) }
 
-  val trip =
-      tripsViewModel.selectedTrip.value
-          ?: return Text(text = "No trip selected. Should not happen", color = Color.Red)
+  // Collect selectedTrip as state to avoid calling .value directly in composition
+  val trip by tripsViewModel.selectedTrip.collectAsState()
+
+  // Check if the selected trip is null and navigate to "Overview" if true
+  if (trip == null) {
```

Comment:
can we make a function to handle this case? I think we can use it in other parts of the code where we assume that the trip is not null